src/components/common/DialogComponent.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Button, Dialog, Portal, CloseButton } from '@chakra-ui/react';
```

Comment:
💊 `as`를 사용하여 Chakra UI에서 제공하는 Dialog를 다른 이름으로 import하고, 우리의 Dialog 컴포넌트를 `Dialog` 이름 그대로 사용해봐요!
```suggestion
import { Button, Dialog as ChakraDialog, Portal, CloseButton } from '@chakra-ui/react';
```